packages/mobile/src/components/core/Screen.tsx

```diff
@@ -6,6 +6,13 @@ import { Animated, StyleProp, View, ViewStyle } from 'react-native'
 
 import { makeStyles } from 'app/styles'
 
+const removeFalsy = (obj: Record<string, any>) =>
```

Comment:
Lodash `pickBy` would be useful here. Also this is checking specifically for undefined, so the name could be more specific

Reply:
ah thank you! i was tryna figure out which lodash thing would work here. and totally agreed!